### PR TITLE
Add test for negative Tx validity when using simple script

### PR DESCRIPTION
**Pull request by mkoura**

### CODE REVIEW

File: cardano_node_tests/tests/test_scripts.py [hunk 1]
```diff
@@ -1245,6 +1246,78 @@ def test_tx_missing_validity(
...
+        payment_addrs: List[clusterlib.AddressRecord],
+        use_build_cmd: bool,
+    ):
+        """Check that it is NOT possible to spend from script address when validity is negative."""
+        temp_template = f"{common.get_test_id(cluster)}_{use_build_cmd}"
+
+        payment_vkey_files = [p.vkey_file for p in payment_addrs]
+        payment_skey_files = [p.skey_file for p in payment_addrs]
+
+        # create multisig script
```
Comment (saratomaz) [inline cardano_node_tests/tests/test_scripts.py:1263]: we should decide if we want to start the comments with capital letter or not :)

**Review by saratomaz — APPROVED** [review]
